playwright/async_api/_context_manager.py

```diff
@@ -33,9 +33,15 @@ async def __aenter__(self) -> AsyncPlaywright:
         loop = asyncio.get_running_loop()
         self._connection._loop = loop
         loop.create_task(self._connection.run())
-        playwright = AsyncPlaywright(
-            await self._connection.wait_for_object_with_known_name("Playwright")
+        done, pending = await asyncio.wait(
```

Comment:
whatever you use remote server or local pipe service, the `PipeTransport` will always started and this code will not work, so maybe we should give a option for not start the local process when use remote endpoint?

Reply:
yep would be nice but so far not doable, because BrowserType depends currently on the rpc channel logic.

Reply:
I want to move this in `launch`, so we can wait the `Playwright` and the transport connection error in `launch` and `connect`

Reply:
I can't just move to `launch`, because lots things depends on `Channel`, maybe we should re-design `PlaywrightImpl` and `PlaywrightContextManager`, allow channel can be inited after `connect` or `launch`.

Reply:
fixed by catch the error in `connect`